src/test.tsx

```diff
@@ -0,0 +1,216 @@
+import { useState, useEffect } from 'react'
+
+interface User {
+  id: number
+  name: string
+  status: any
```

Comment:
Não é recomendado que usemos any, isso aqui é um codesmell e o ideal é que você informe o tipo, exemplo:

```suggestion
  status: 'active' | 'inactive'
```